GVFS/GVFS.UnitTests/Mock/Common/MockPlatform.cs

```diff
@@ -107,6 +107,16 @@ public override string GetDataRootForGVFSComponent(string componentName)
             return Path.Combine(this.GetDataRootForGVFS(), componentName);
         }
 
+        public override string GetCommonAppDataRootForGVFS()
+        {
+            return this.GetDataRootForGVFS();
+        }
+
+        public override string GetLogsDirectoryForGVFSComponent(string componentName)
+        {
+            return Path.Combine(this.GetCommonAppDataRootForGVFS(), componentName);
+        }

```

Comment:
Should `GetLogsDirectoryForGVFSComponent` be updated to better match the behavior of VFS4G on Windows (the log directory not being a subdirectory of `GetCommonAppDataRootForGVFS`)?

Reply:
If it's not too much work this might be a good time to also address the TODO in `GetDataRootForGVFS`:

```// TODO: Update this method to return non existant file path.```

Reply:
>>Should GetLogsDirectoryForGVFSComponent be updated to better match the behavior of VFS4G on Windows (the log directory not being a subdirectory of GetCommonAppDataRootForGVFS)?

Make sense. [Updated](https://github.com/microsoft/VFSForGit/pull/1597/commits/462ab783797119f076c8a9578f85deb284a14108).

Reply:
>>If it's not too much work this might be a good time to also address the TODO in GetDataRootForGVFS:

Let me take a look.

Reply:
This could be done. [Updated](https://github.com/microsoft/VFSForGit/pull/1597/commits/6776623ed0c0605871acd9911ee60a65da67cc8a)